taskcluster/kinds/docker-image/kind.yml

```diff
@@ -22,21 +22,75 @@ tasks:
         args:
             PYTHON_VERSION: "3.8"
             PYTHON_REQ_SUFFIX: ".py38"
-    pushapkscript-python38:
-        definition: pushapkscript
-        parent: python38
-    pushapkscript-python39:
-        definition: pushapkscript
-        parent: python39
-    pushflatpakscript-python38:
-        definition: pushflatpakscript
-        parent: python38
-    pushflatpakscript-python39:
-        definition: pushflatpakscript
-        parent: python39
-    signingscript-python38:
-        definition: signingscript
-        parent: python38
-    signingscript-python39:
-        definition: signingscript
-        parent: python39
+
+    # Used by push-image kind
+    skopeo: {}
+
+    base:
+        args:
+            BASE_IMAGE: python:3.9.7
+    # TODO: Figure out PYTHON_REQ_SUFFIX when we decide to have py3.10
+    # base-3.10:
+    #     definition: base
+    #     args:
+    #         BASE_IMAGE: python:3.10
+    addonscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: addonscript
```

Comment:
optional suggestion: These could be set in a transform file that runs before the docker-image transforms.